Changelog.md

```diff
@@ -2,6 +2,17 @@
 
 ### Gazebo Physics 6.x.x (202X-XX-XX)
 
+### Gazebo Physics 6.3.2 (2023-03-28)
+
+1. Fix joint index used in bookkeeping
+    * [Pull request #498](https://github.com/gazebosim/gz-physics/pull/498)
+
+1. Add main function to joint transmitted wrench test
+    * [Pull request #496](https://github.com/gazebosim/gz-physics/pull/496)
+
+1. Move joint transmitted wrench test to separate file
+    * [Pull request #495](https://github.com/gazebosim/gz-physics/pull/495)
```

Comment:
I would add #496 to this entry

Reply:
https://github.com/gazebosim/gz-physics/pull/499/commits/94bbc83cc37e51bddd125ff4477828fbd69db20b